### PR TITLE
Add devise

### PR DESCRIPTION
**Pull request by Enoisong**
Dear Reviewer,

In this PR we have done the following👨‍💻👨‍💻

## Requirements
-  Remove current_user method in ApplicationController🤸‍♂️🤸‍♂️
-  [Install and setup devise](https://github.com/heartcombo/devise#getting-started).
    -   Can register a new user✏
    -   User logs in with a combination of email and password.🗝
    -   Hashed password should be stored in the database.🐚🐚
    -   Ask for confirmation of email.🧍‍♂️
    -   Can reset password🗝
[Modify the views in devise](https://github.com/heartcombo/devise#configuring-views) for registration and login to match [the wireframes from the Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A) and  styling.🤸‍♂️🤸‍♂️

## Achievements
-  Remove current_user method in ApplicationController🤸‍♂️✅
-  [Install and setup devise](https://github.com/heartcombo/devise#getting-started)🐚✅
    -   Can register a new user✏
    -   User logs in with a combination of email and password.🗝
    -   Hashed password should be stored in the database.🐚🐚
    -   Ask for confirmation of email.🧍‍♂️
    -   Can reset password🗝
[Modify the views in devise](https://github.com/heartcombo/devise#configuring-views) for registration and login to match [the wireframes from the Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A) and  styling.🤸‍♂️✅

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (VICTOR-CHUKWU) [inline app/models/user.rb]: - [ ] kindly allow the user to confirm his/her email while registering by adding confirmable :+1:

File: app/views/devise/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```
Comment (VICTOR-CHUKWU) [inline app/views/devise/sessions/new.html.erb:25]: - [ ] kindly make your design match the wireframe
your design :point_down: 
image
wireframe given :point_down: 
Screenshot from 2023-09-06 19-26-16

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Enoisong  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- devise correctly installed and used


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Enoisong** [comment]
## Required Changes Implementation👨‍💻👨‍💻

## Requirements
-   kindly allow the user to confirm his/her email while registering by adding confirmable 👍
-   kindly make your design match the wireframe
 

## Achievements

Add `confirmable` that allows the user to confirm his/her email while registering.💯✔

All Implementations of the design matches the `sneak peak` wireframes💯✔

**Review by mikethreels — APPROVED** [review]
Hi Enobong, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job following the instructions of the previous CR.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**